frontend/src/features/workspace/components/DeleteFormModal/DeleteFormModal.tsx

```diff
@@ -0,0 +1,110 @@
+import { useCallback } from 'react'
+import { BiFileBlank } from 'react-icons/bi'
+import {
+  ButtonGroup,
+  Container,
+  Icon,
+  ListItem,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  OrderedList,
+  Text,
+  useBreakpointValue,
+  UseDisclosureReturn,
+} from '@chakra-ui/react'
+
+import { AdminDashboardFormMetaDto } from '~shared/types'
+
+import Button from '~components/Button'
+import { ModalCloseButton } from '~components/Modal'
+
+import { useDeleteFormMutation } from '~features/workspace/mutations'
+
+export interface DeleteFormModalProps
+  extends Pick<UseDisclosureReturn, 'onClose' | 'isOpen'> {
+  formToDelete?: Pick<AdminDashboardFormMetaDto, '_id' | 'title' | 'admin'>
+}
+
+export const DeleteFormModal = ({
+  isOpen,
+  onClose,
+  formToDelete,
+}: DeleteFormModalProps): JSX.Element | null => {
+  const modalSize = useBreakpointValue({
+    base: 'mobile',
+    xs: 'mobile',
+    md: 'md',
+  })
+
+  const { deleteFormMutation } = useDeleteFormMutation()
+
+  const handleDeleteForm = useCallback(() => {
+    if (!formToDelete) return
+    return deleteFormMutation.mutate(formToDelete._id, {
+      onSuccess: onClose,
+    })
+  }, [deleteFormMutation, formToDelete, onClose])
+
+  if (!formToDelete) return null
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} size={modalSize}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalCloseButton />
+        <ModalHeader color="secondary.700">
+          <Container maxW="42.5rem" p={0}>
+            Delete form
+          </Container>
+        </ModalHeader>
+        <ModalBody whiteSpace="pre-line">
+          <Text color="secondary.500">
+            You will lose all responses and feedback for the following form
+            permanently. Are you sure you want to delete the form?
+          </Text>
+          <OrderedList
+            spacing="0.5rem"
+            listStyleType="none"
+            ml="1.75rem"
+            mt="1rem"
+          >
```

Comment:
just a thought: is there a reason why we use `OrderedList` here? seems a bit strange to use it for just one item haha. nts if doing a `Grid` would be more natural but achieve the same effect, nbd tho

Reply:
I'll probably change to a normal unordered list, but this is preoptimizing for future bulk delete hehe

Reply:
1602960